app/controllers/jobs.controller.js

```diff
@@ -54,13 +61,22 @@ async function returnLogs(request, h) {
     return h.response().code(NOT_FOUND_STATUS_CODE)
   }
 
+  let licenceReference
+
   ProcessReturnLogsService.go(cycle)
 
+  if (h.request.payload !== null && h.request.payload.licenceReference) {
```

Comment:
I think this might be a merge issue ? 

i am not in the git blame @robertparkinson

Reply:
yeah that should't be there